lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_nas_procedures.cpp

```diff
@@ -996,19 +1005,21 @@ void s1ap_handle_conn_est_cnf(
         conn_est_cnf_pP->e_rab_level_qos_preemption_vulnerability[item];
 
     if (conn_est_cnf_pP->nas_pdu[item]) {
-      S1ap_NAS_PDU_t* nas_pdu = calloc(1, sizeof(S1ap_NAS_PDU_t));
+      S1ap_NAS_PDU_t* nas_pdu =
+          reinterpret_cast<S1ap_NAS_PDU_t*>(calloc(1, sizeof(S1ap_NAS_PDU_t)));
       nas_pdu->size = blength(conn_est_cnf_pP->nas_pdu[item]);
-      nas_pdu->buf = malloc(blength(conn_est_cnf_pP->nas_pdu[item]));
+      nas_pdu->buf = reinterpret_cast<uint8_t*>(
+          malloc(blength(conn_est_cnf_pP->nas_pdu[item])));
```

Comment:
```suggestion
          calloc(blength(conn_est_cnf_pP->nas_pdu[item])));
```

Reply:
@rsarwad , it will be good to fix this even though it is not part of the changes in this PR.